### DISCUSSION TITLE
Version

### DISCUSSION THREAD

**Discussion by Hubobel** [mealie-recipes/mealie#3502] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

There is no way to identify the installed version in the frontend?

### Please provide a concise description of the feature that would resolve your issue.

.

### Please consider and list out some caveats or tradeoffs made in your design decision

.

### Additional Information

- [X] If this is accepted I'm willing to submit a PR to provide this feature
- [X] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by boc-the-git** [comment]
BASEURL/admin/site-settings

The 'Bug Report' button will give you an easy way to get all the data too.
